src/graphorm/graphorm.ts

```diff
@@ -338,7 +344,26 @@ export class GraphORM {
       newBuilder = this.mappings[type].additionalQuery(ctx, alias, newBuilder);
     }
     (this.mappings?.[type]?.requiredColumns ?? []).forEach((col) => {
-      newBuilder = newBuilder.addSelect(`${alias}."${col}"`, col);
+      const columnOptions =
+        typeof col === 'object'
+          ? col
+          : {
+              column: col,
+              columnAs: col,
+              isJson: false,
+            };
+
+      if (columnOptions.isJson) {
```

Comment:
Why do we need if/else, it looks the same?

Reply:
nvm i understand now. the if part allows sql injection (kinda)

Reply:
column brackets break with `->` eg. `alias.""contentMeta"->'translate_title'->'translations'"`, and with normal column name its ok `alias."title"`

Reply:
yeah, I initially wanted to do `includes('->')` but I think with `isJson` is more safe and we have to consciously in code say this is json column